chai-as-promised/.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
```

Comment:
There has been more concern about [enforcing a standard project wide](https://github.com/borisyankov/DefinitelyTyped/issues/1569#issuecomment-35524237), but this ensures the rules are only set folder-wide.